students/byeongyeon/westagram/users/views.py

```diff
@@ -1,3 +1,35 @@
-from django.shortcuts import render
-
 # Create your views here.
+from django.shortcuts       import render
+import json
+
+from django.http            import JsonResponse
+from django.views           import View
+
+from users.models           import User
+from users.validator        import validate_email, validate_password, validate_mobile
```

Comment:
사용하지 않는 모듈은 제거해 주세요.
```suggestion
import json

from django.http     import JsonResponse
from django.views    import View

from users.models    import User
from users.validator import validate_email, validate_password, validate_mobile
```